src/library/returnBook/ReturnBookControl.java

```diff
@@ -0,0 +1,82 @@
+package library.returnBook;
+import library.entities.Book;
+import library.entities.Library;
+import library.entities.Loan;
+
+public class ReturnBookControl {
+
+    private ReturnBookUI Ui;

```

Comment:
"Ui" should be all lowercase to my understanding.

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/5/commits/d8299028e4cec7508480a37efec009c391a18e58